src/berry_mill/kiwrap.py

```diff
@@ -107,9 +122,17 @@ def _get_repokeys(self, reponame: str, repodata: Dict[str, str]) -> str:
         else:
             s_url = urljoin(f"{url.scheme}://{url.netloc}/{url.path}/Release.key", "")
             response = requests.get(s_url, allow_redirects=True)
-            with open(g_path, 'xb') as f_rel:
-                f_rel.write(response.content)
-            response.close()
+            # check reponse OK
+            if response.status_code == 200:
```

Comment:
It should be `HTTPStatus.OK` instead, taken from `http` library.

Reply:
done